cypress/e2e/page/annotation-pagetests.cy.js

```diff
@@ -38,11 +38,17 @@ describe("tests on annotation page ui with programmatic state loading and store
                 cy.window().its("$nuxt.$router").then(router => {
 
                     router.push({ path: "/annotation" });
+                    cy.commitToVuexStore("setCurrentPage", "annotation");
                 });
             });
 
             it("Annotate age column; default age format transformations", () => {
 
+                console.log("Start of test store:");
```

Comment:
What is the purpose of this section? If I understand right, it prints the contents of the store to console. Unless I misunderstand this should have no impact on the running of the tests, so let's remove it.